www/lobby/id.ts

```diff
@@ -1,16 +1,42 @@
-import { Router, Request, Response } from 'express';
+import { Router, Request, Response, NextFunction } from 'express';
+// import { nextTick } from 'process';
```

Comment:
remove this

Reply:
Good catch

> thanks for the changes! out of curiosity, why the file change from `:id.tsx` => `id.tsx`

My vscode doesn't recognize files that contain the semicolon character in the name, so I thought it would be easier to not have to deal with this problem since naming the file :id.ts vs id.ts doesnt seem to be too important